modules/tile-converter/src/converter-cli.ts

```diff
@@ -308,6 +308,20 @@ function getStringValue(index: number, args: string[]): string {
   return value;
 }
 
+/**
+ * Modyfy input tileset path to be compatible with fetch
+ * @param index - option's name index in the argument's array.
+ *                The value of the option should be next to name of the option.
+ * @param args - cli arguments array
+ * @returns - string value of the option
+ */
+function getTilesetValue(index: number, args: string[]): string {
```

Comment:
Is this just `getURLValue()` or is there something more tileset specific happening here?